{"metadata":{"externalID":"s:21IdentityOrchestration17TSIdoJourneyErrorO07networkE0yA2CmF","roleHeading":"Case","modules":[{"name":"IdentityOrchestration"}],"role":"symbol","title":"TSIdoJourneyError.networkError","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"networkError"}],"symbolKind":"case"},"schemaVersion":{"minor":3,"patch":0,"major":0},"kind":"symbol","abstract":[{"type":"text","text":"Could not connect to server, or server did not respond before timeout."}],"sections":[],"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"networkError","kind":"identifier"}]}]}],"identifier":{"url":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError\/networkError","interfaceLanguage":"swift"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/identityorchestration\/tsidojourneyerror\/networkerror"]}],"hierarchy":{"paths":[["doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration","doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError"]]},"references":{"doc://com.TransmitSecurity.IdentityOrchestration/documentation/IdentityOrchestration/TSIdoJourneyError":{"fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"TSIdoJourneyError","kind":"identifier"}],"kind":"symbol","navigatorTitle":[{"text":"TSIdoJourneyError","kind":"identifier"}],"abstract":[{"type":"text","text":"The enum for the start Journey errors."}],"title":"TSIdoJourneyError","identifier":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError","url":"\/documentation\/identityorchestration\/tsidojourneyerror","type":"topic","role":"symbol"},"doc://com.TransmitSecurity.IdentityOrchestration/documentation/IdentityOrchestration/TSIdoJourneyError/networkError":{"abstract":[{"text":"Could not connect to server, or server did not respond before timeout.","type":"text"}],"url":"\/documentation\/identityorchestration\/tsidojourneyerror\/networkerror","title":"TSIdoJourneyError.networkError","kind":"symbol","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"networkError","kind":"identifier"}],"identifier":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration\/TSIdoJourneyError\/networkError","type":"topic"},"doc://com.TransmitSecurity.IdentityOrchestration/documentation/IdentityOrchestration":{"abstract":[],"type":"topic","url":"\/documentation\/identityorchestration","kind":"symbol","role":"collection","title":"IdentityOrchestration","identifier":"doc:\/\/com.TransmitSecurity.IdentityOrchestration\/documentation\/IdentityOrchestration"}}}